### PR TITLE
Skeleton table improvements

### PR DESCRIPTION
**Pull request by leSamo**
- Show active (blue) sorting indicator on skeleton table
- Show expandable button on skeleton table rows
- Fix margin between table toolbar and filter chips changing during table load

### CODE REVIEW

File: src/Components/SmartComponents/Table/BaseTableToolbar.js [hunk 1]
```diff
@@ -18,7 +18,11 @@ const BaseTableToolbar = ({
     <PrimaryToolbar
       pagination={
         isLoading ? (
-          <Skeleton fontSize="xl" width="200px" style={{ margin: 10 }} />
+          <Skeleton
+            fontSize="xl"
+            width="200px"
+            style={{ marginTop: 8, marginRight: 16 }}
```
Comment (mkholjuraev) [inline src/Components/SmartComponents/Table/BaseTableToolbar.js:24]: Why do we need this specific margins? would not it result in discrepancy with other apps? if not, can we use pf variables for this purpose?

Comment (leSamo) [reply src/Components/SmartComponents/Table/BaseTableToolbar.js]: You are right, I removed these margins and now it's consistent with Inventory and other apps

File: package.json [hunk 2]
```diff
@@ -51,7 +51,6 @@
     "@babel/preset-env": "^7.24.7",
     "@babel/preset-react": "^7.24.7",
     "@cypress/code-coverage": "^3.12.36",
-    "@cypress/react18": "^2.0.0",
```
Comment (gkarat) [inline package.json:54]: This must stay and, in cypress/support/component.js, replace 'cypress/react'. However, do we want to do this change in this unrelated commit/PR?

**Review by gkarat — APPROVED** [review]
LGTM, thanks @leSamo, just one minor comment

File: package.json (hunk 2 above)
Comment (leSamo) [reply package.json:54]: You're right, this was removed accidentally, nice catch
